### ISSUE TITLE
7360v2: no configmode on half of the ports

### ISSUE THREAD

**Issue by AiyionPrime** [freifunk-gluon/gluon#2805]
### Bug report

**What is the problem?**

- What is not working as expected?

The Config Mode is not reachable on the ports three and four, but is on one and two.

- How is it misbehaving?

The Ports do work in normal operation, but not in Config Mode.

- When did the problem first start showing up?

The problem appears in current master and existed in [v2021.1.2](https://github.com/freifunk-gluon/gluon/tree/v2021.1.2) as well.

Possible the faulty behavior was just not spotted in its introduction to gluon in #1889.

- What were you doing when you first noticed the problem?

@bschelm was entering the Config Mode.
@grische and me verified the problem on a different device.

- On which devices (vendor, model and revision) is it misbehaving?

AVM FritzBox 7360 v2

- Does the issue appear on multiple devices or targets?

Haven't tested it on v1/SL, but would be worth a shot; the devices are not expected to differ much, excluding the memory.

**What is the expected behaviour?**
The configmode should work regardless of the driver the ports use.
Unlikely it ever worked on both ports.

**Gluon Version:**

master, gluon-v2022.1-268-g4484426

**Site Configuration:**

https://github.com/freifunkh/site/commit/58842a3ad2750b58d84d77978f17da507c48f6fb

**Custom patches:**
Three, but none related to lantiq, or even the device:

[0001-fix-when-vlan-crc-is-00000000.patch](https://github.com/freifunkh/site/blob/master/patches/0001-fix-when-vlan-crc-is-00000000.patch)
[0002-patches-uradvd-change-ip-lifetimes-to-150-300-seco](https://github.com/freifunkh/site/blob/master/patches/0002-patches-uradvd-change-ip-lifetimes-to-150-300-second.patch)
[0003-patches-openwrt-add-TL-WR841ND-N-8M-and-16M-variants.patch](https://github.com/freifunkh/site/blob/master/patches/0003-patches-openwrt-add-TL-WR841ND-N-8M-and-16M-variants.patch)

**Note**

as @grische found, ports one and two have a different driver:

```console
[    1.491010] gswip 1e108000.switch lan3 (uninitialized): PHY [1e108000.switch-mii:00] driver [Qualcomm Atheros AR8030] (irq=POLL)
[    1.504519] gswip 1e108000.switch lan4 (uninitialized): PHY [1e108000.switch-mii:01] driver [Qualcomm Atheros AR8030] (irq=POLL)
[    1.520087] gswip 1e108000.switch lan2 (uninitialized): PHY [1e108000.switch-mii:11] driver [Intel XWAY PHY11G (xRX v1.2 integrated)] (irq=POLL)
[    1.536986] gswip 1e108000.switch lan1 (uninitialized): PHY [1e108000.switch-mii:13] driver [Intel XWAY PHY11G (xRX v1.2 integrated)] (irq=POLL)
```

**Comment by Djfe** [comment]
I tested 7360SL and 7360 V2 on v2022.1.4 and on master
Config mode is working on all ports for me

**Comment by neocturne** [comment]
@AiyionPrime Do you know if this is still an issue with current Gluon main?

**Comment by fsffunk** [comment]
@neocturne 
No, the issue described here is not there with current firmware release i tested https://github.com/freifunk-gluon/gluon/issues/3525

**Comment by grische** [comment]
I was able to reproduce the issue with the same box and Gluon v2023.2.5. 

@AiyionPrime can we reopen the issue?

@fsffunk are you sure you had a V2 model? Can you confirm it on multiple devices?

**Comment by AiyionPrime** [comment]
@fsffunk appears to be unavailable?

I can ask @bschelm to lend me the v2 again and verify the problems existence on `v2023.2.5` and `main` on Thursday, if that would help.

@Djfe would you mind verifying `v2023.2.5` is broken on your device as well?

**Comment by fsffunk** [comment]
@grische 
I can now confirm this issue with a second 7360v2. One 7360v2 have this issue, an other one did not.

I can test now the 7360v2(Art. Nr.: 2000 2522 flashed with https://github.com/maurerle/fritz-tools , auto detected its bootloader as 7360v2 in the tool, listed as v2 in here: https://www.router-faq.de/?id=fbinfo&hwf=fb7360 ) with the issue some more. Or even donate it if someone have the intention to fix the issue and when fixed run it somewhere 24/7 creating public wlan hotspot.

The 7360v2 with the issue:
Image: https://firmware.freifunk-aachen.de/firmware/download/from-2025.1.x/sysupgrade/gluon-ffac-v2025.0.0-12-avm-fritz-box-7360-v2-sysupgrade.bin

When in config mode, you get the ipv4 address on LAN1 and LAN2. LAN3 and LAN4 are dead.
After the configuration, and reboot, you have your classic LAN1 as WAN and LAN2 where you get your IPv6 while offline. LAN3 and LAN4 still dead.
Also: The USB-WAN-Hotplug mode did not seem to work there. Even after about 15minutes of waiting there was no ipv4/internet address on LAN2 port.

Installed back again this image: https://downloads.openwrt.org/releases/25.12.0-rc1/targets/lantiq/xrx200/openwrt-25.12.0-rc1-lantiq-xrx200-avm_fritz7360-v2-squashfs-sysupgrade.bin

LAN3 and LAN4 ports came back to life with latest OpenWrt image.
